Configurations/10-main.conf

```diff
@@ -1730,8 +1766,9 @@ my %targets = (
 ##### VxWorks for various targets
     "vxworks-ppc60x" => {
         inherit_from     => [ "BASE_unix" ],
-        cc               => "ccppc",
-        cflags           => "-mrtp -mhard-float -mstrict-align -fno-implicit-fp -O2 -fstrength-reduce -fno-builtin -fno-strict-aliasing -Wall",
+        CC               => "ccppc",
+        CFLAGS           => "-O2 -Wall",
+        cflags           => "-mrtp -mhard-float -mstrict-align -fno-implicit-fp -fstrength-reduce -fno-builtin -fno-strict-aliasing",
```

Comment:
One can wonder which of these belong in CFLAGS, but we probably don't care, right? But at least -fstrength-reduce is an optimization option. Shug! Side note.

Reply:
Changed